restock.lic

```diff
@@ -47,13 +48,16 @@ class Restock
         reget(2, 'runestone') ? DRCI.put_away_item?(item['name'], @runestone_storage) : stow_hands
       end
     end
+    deposit_coins_from_specific_bank?(@keep_copper, @settings, town)
   end
 
   def setup
     @settings = get_settings
     @restock = @settings.restock
     @hometown = @settings.hometown
     @runestone_storage = @settings.runestone_storage
+    amount, denom = @settings.sell_loot_money_on_hand.split(' ')
+    @keep_copper = convert_to_copper(amount, denom)
```

Comment:
I think this can be simplified by just storing `sell_loot_money_on_hand` and making only one call to `withdraw_exact_amount` in `deposit_coins_from_specific_bank`

Reply:
I may not need `deposit_coins_from_specific_bank?`. I was following what was put in to make it work with restock but see the original had an option already but defaults to `settings.hometown` if nil. Testing with the old. I will look into this.